src/js/components/modal.js

```diff
@@ -0,0 +1,29 @@
+export class cookies {
+    $bodyElement = document.querySelector('body');
+    bodyOverlayClassName = 'overlay';
+    hideClassName = 'hide';
+    showModalClassName = 'show';
+    
+
+    constructor(modalName) {
+        this.modal = document.querySelector(`#${modalName}`);
+        this.closeButton = this.modal.querySelector('button');
+        this.localStorage = localStorage.getItem("cookiesAccepted");
+    }
+
+    handleCloseButton() {
+
+        this.closeButton.addEventListener('click', function() {
+            localStorage.setItem("cookiesAccepted", "accepted2");
+            if(this.localStorage = "cookiesAccepted") {
+                this.modal.classList.add(this.hideClassName);
+                this.$bodyElement.classList.remove(this.bodyOverlayClassName);
+            }
+        })
+    }
+
+    init() {
+        this.$bodyElement.classList.add(this.bodyOverlayClassName);
```

Comment:
te 2 linijki owijasz w if-a, i sprawdzasz w nim this.checkLocalStorage(), dzieki temu bedzie Ci sie pojawiac ta ciemna poswiata albo nie

jedyne co trzeba bedzie dorobic jescze to klase w css, ktora bedzie pokazywac lub ukrywac modal, ale to juz mozna potraktowac jako EXTRA zadanie, wazne zeby wszystko pozostale dzialalo